lib/eth/EthRpc.js

```diff
@@ -118,10 +120,18 @@ class EthRPC {
     const result = [];
     for (const [i, request] of payToArray.entries()) {
       const { address, amount } = request;
+      const emitData = {
+        address: request.address,
+        amount: request.amount,
+        id: request.id
+      };
+      this.emitter.emit('attempt', emitData);
       try {
         const txid = await this.sendToAddress({ address, amount, passphrase });
```

Comment:
I wonder if we should emit the `unlock` event in sendToAddress if they provide a passphrase